woocommerce/woocommerce
# Add additional body class for the account dashboard (#30050)

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This adds a body class of `woocommerce-dashboard` to the account home page, for the reasons highlighted in #30050.

Closes #30050.



### How to test the changes in this Pull Request:

1. Use the Storefront theme
2. Ensure you are logged in to the frontend as a customer
3. Access the my account page (don't click any of the tabs)
4. Review the body classes on this page, you'll see an additional `woocommerce-dashboard` class (i've used this name because the default tab is labelled Dashboard)
5. Click into the other tabs of my account, on these pages you'll see that the body class is not included
6. Log out, on the login page for my account, you'll see that the body class is not included



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Add additional body class for the account dashboard

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
